layouts/partials/footer.html

```diff
@@ -1,21 +1,43 @@
 {{ $copy := site.Copyright }}
 {{ $year := now.Year }}
 {{ $logo := printf "img/logos/%s" site.Params.logos.footer | relURL }}
-<footer class="footer has-background-black">
+<footer class="footer has-background-white">
   <div class="container">
     <div class="columns">
       <div class="column">
         <img class="is-footer-logo" src="{{ $logo }}" alt="{{ site.Title }} footer logo">
       </div>
 
       <div class="column">
-        
+        <p class="is-size-6 has-text-black has-text-weight-light">
+          <a class="has-text-black is-small" target="_blank" href="{{ site.Params.github }}">
+            <i class="fab fa-github"></i>&nbsp;GitHub
+          </a>
+        </p>
+        <p class="is-size-6 has-text-black has-text-weight-light">
+          <a class="has-text-black is-small" target="_blank" href="{{ site.Params.logo_repo }}">
+            <i class="fa fa-palette"></i>&nbsp;Logos
+          </a>
+        </p>
+      </div>
+
+      <div class="column">
+        <p class="is-size-6 has-text-black has-text-weight-light">
+          <a class="has-text-black is-small" target="_blank" href="{{ site.Params.slack }}">
+            <i class="fab fa-slack"></i>&nbsp;Slack
+          </a>
+        </p>
+        <p class="is-size-6 has-text-black has-text-weight-light">
+          <a class="has-text-black is-small" target="_blank" href="{{ site.Params.twitter }}">
+            <i class="fa fa-palette"></i>&nbsp;Twitter
+          </a>
+        </p>
       </div>
     </div>
 
     <hr />
 
-    <p class="is-size-6 has-text-light has-text-weight-light">
+    <p class="is-size-6 has-text-black has-text-weight-light">
```

Comment:
Black is the default, so you don't need to specify that.

Reply:
Seems like some sort of grey is default, not black.